db/schema.rb

```diff
@@ -49,6 +49,7 @@
     t.datetime "updated_at"
     t.datetime "applied_at"
     t.integer  "updater_id"
+    t.text     "state",                       default: "draft", null: false
```

Comment:
I decided to go with `state` because a few of the controllers/helpers depend on a `state` method/attribute being available, and `aasm` doesn't care either way.

But sure, I can rename it, and add a `state` that delegates to `aasm_state`. Or would you prefer to rename the calls as well?